blockchain/serviceMetadata_test.go

```diff
@@ -1,40 +1,39 @@
 
 func TestAllGetterMethods(t *testing.T) {
-	config.Vip().Set(config.FreeCallEndPoint,"http://demo8325345.mockable.io")
+	config.Vip().Set(config.FreeCallEndPoint, "http://demo8325345.mockable.io")
```

Comment:
BTW, it is not related to this PR, but can this test be executed offline (without internet access)?

Reply:
no , but let me explore on alternatives , we can create an issue to track this

Reply:
Yes, lets create issue on this. Once I tried to compile daemon on the plane board and found that it doesn't compiles (the problem was in my test of Ethereum functionality). Since that moment I think that offline compilation is a good thing :-)

Reply:
Sure, done #429